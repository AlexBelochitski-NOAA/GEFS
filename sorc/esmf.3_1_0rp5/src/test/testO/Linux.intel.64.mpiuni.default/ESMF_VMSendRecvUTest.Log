20190610 112233.519426 INFO     PET0 Running with ESMF Version 3.1.0rp5
20190610 112233.519689 INFO     PET0  Beginning Test, file ESMF_VMSendRecvUTest.F90, line          80
20190610 112233.519706 INFO     PET0  NUMBER_OF_PROCESSORS           1
20190610 112233.519760 INFO     PET0  PASS  Sending local data Test, ESMF_VMSendRecvUTest.F90, line         151
20190610 112233.519786 INFO     PET0  PASS  Verify local data after receive Test, ESMF_VMSendRecvUTest.F90, line         162
20190610 112233.522651 INFO     PET0  PASS  Sending local data Test, ESMF_VMSendRecvUTest.F90, line         172
20190610 112233.522719 INFO     PET0  PASS  Verify local data after receive Test, ESMF_VMSendRecvUTest.F90, line         184
20190610 112233.522728 INFO     PET0  PASS  Sending local data Test, ESMF_VMSendRecvUTest.F90, line         194
20190610 112233.522796 INFO     PET0  PASS  Verify local data after receive Test, ESMF_VMSendRecvUTest.F90, line         206
20190610 112233.522813 INFO     PET0  PASS  Sending local data Test, ESMF_VMSendRecvUTest.F90, line         216
20190610 112233.522920 INFO     PET0  PASS  Verify local data after receive Test, ESMF_VMSendRecvUTest.F90, line         237
20190610 112233.522926 INFO     PET0  Number of failed tests:           0
20190610 112233.522940 INFO     PET0  Ending Test, file ESMF_VMSendRecvUTest.F90, line         239
