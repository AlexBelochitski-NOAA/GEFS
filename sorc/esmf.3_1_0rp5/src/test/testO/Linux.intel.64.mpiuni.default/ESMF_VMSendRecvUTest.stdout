  Beginning Test, file ESMF_VMSendRecvUTest.F90, line          80
  NUMBER_OF_PROCESSORS           1
  PASS  Sending local data Test, ESMF_VMSendRecvUTest.F90, line         151
           0  After rcv i_recvData is            1           2 ( should be 
           1           2  )
  PASS  Verify local data after receive Test, ESMF_VMSendRecvUTest.F90, line    
      162
  PASS  Sending local data Test, ESMF_VMSendRecvUTest.F90, line         172
           0 After recv r4_recvData is    1.000000       2.000000     
 ( should be    1.000000       2.000000      )
  PASS  Verify local data after receive Test, ESMF_VMSendRecvUTest.F90, line    
      184
  PASS  Sending local data Test, ESMF_VMSendRecvUTest.F90, line         194
           0 After recv r8_recvData is    1.00000000000000     
   2.00000000000000      ( Should be    1.00000000000000     
   2.00000000000000       )
  PASS  Verify local data after receive Test, ESMF_VMSendRecvUTest.F90, line    
      206
  PASS  Sending local data Test, ESMF_VMSendRecvUTest.F90, line         216
           0 After recv: Recv_Logical(1) is False
           0 After recv: Recv_Logical(2) is True
           0 After recv: Logical_soln(1) is False
           0 After recv: logical_soln(2) is True
  PASS  Verify local data after receive Test, ESMF_VMSendRecvUTest.F90, line    
      237
  Number of failed tests:           0
  Ending Test, file ESMF_VMSendRecvUTest.F90, line         239
